addons/event/views/event_views.xml

```diff
@@ -354,6 +362,10 @@
                                         </t>
                                     </h4>
                                 </div>
+                                <div class="o_event_bottom_right">
+                                    <field name="kanban_state" widget="state_selection" groups="base.group_user"/>
```

Comment:
Why specifying group user ? Portal / public users have no access to kanban views unless I am mistaken.